testbed/testbed/receivers.go

```diff
@@ -254,9 +254,8 @@ func (zr *ZipkinDataReceiver) Start(tc *MockTraceConsumer, _ *MockMetricConsumer
 	cfg := factory.CreateDefaultConfig().(*zipkinreceiver.Config)
 	cfg.SetName(zr.ProtocolName())
 	cfg.Endpoint = fmt.Sprintf("localhost:%d", zr.Port)
-
-	params := component.ReceiverCreateParams{Logger: zap.NewNop()}
```

Comment:
All changes in this file can be reverted.

Reply:
reverted